__tests__/app.test.js

```diff
@@ -195,21 +196,69 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 
-  test("400: responds with an error message if searched with wrong url. endpoint /api/articles/:article_id/comments", () => {
+  test("400: responds with an error message if searched with wrong url. URL: /api/articles/:article_id/comments", () => {
     return request(app)
       .get("/api/articles/xyz/comment")
       .expect(400)
       .then(({ body }) => {
         expect(body.msg).toEqual("Bad request. Please check what you're requesting and try again.");
       });
-    });
+  });
+
+  test("404: responds with JSON object of all /api/articles/:article_id/comments endpoints", () => {
+    return request(app)
+      .get("/api/articles/1500/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toEqual("Not Found");
+      });
+  });
+});
+
+//07
+describe("POST /api/articles/:article_id/comments", () => {
+  test("200: responds with a JSON object of the comments of a given article id Which has been posted.", () => {
```

Comment:
If you're expecting a 201 please change the description to match 👍

Reply:
Fixed